packages/js/src/Modules/Verto/webrtc/Call.ts

```diff
@@ -51,6 +51,38 @@ export default class Call extends BaseCall {
     }
   }
 
+  /**
+   * Changes the audio output device (i.e. speaker) used for the call.
+   *
+   * @examples
+   *
+   * Using async/await:
+   *
+   * ```js
+   * await call.setAudioOutDevice('abc123')
+   * ```
+   *
+   * Using ES6 `Promises`:
+   *
+   * ```js
+   * call.setAudioOutDevice('abc123').then(() => {
+   *   // Do something using new audio output device
+   * });
+   * ```
+   *
+   * Usage with {@link BrowserSession.getAudioOutDevices}:
```

Comment:
```suggestion
   * Usage with `{@link TelnyxRTC.getAudioOutDevices}`:
```

I think it is weird to link `BrowserSession` since we don't expose this class directly. Can you confirm that this link to the `TelnyxRTC` instance would work?